src/app.js

```diff
@@ -1 +1,7 @@
 'use strict';
+
+const { copy } = require('./copy');
+
+const [copiedFilePath, newPath] = process.argv.slice(2);
```

Comment:
It is recommended to use more descriptive variable names, like 'sourcePath' instead of 'copiedFilePath'.